README.md

```diff
@@ -1,70 +1,69 @@
-# Getting Started with Create React App
+# Math-Magician
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Available Scripts
+- All the linters has been installed correctly
+- WebPack and React has been set-up correctly
+![Mi video](https://user-images.githubusercontent.com/105079888/188669578-0538c98c-735f-4227-b4d8-811410c9dab7.gif)
 
-In the project directory, you can run:
+> Description the project.
+- Demostrate abilities using WebPack and React.
 
-### `npm start`
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Built With
 
-### `npm test`
+- JavaScript
+- SASS
+- HTML
+- WebPack
+- React
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Live Demo
+[No demo abailable yet]()
 
-### `npm run build`
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+## Getting Started
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+## Setup
+If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+[Download the ZIP from this location](https://github.com/LuisDavidRodriguez/math-magician/archive/refs/heads/development.zip), or run the following [git](https://git-scm.com/downloads)
+ command to clone the files to your machine:
 
-### `npm run eject`
+```bash
+git clone git@github.com:LuisDavidRodriguez/math-magician.git
+```
+- Once the files are on your machine, open the _Portfolio_ folder in [Visual Studio Code](https://code.visualstudio.com/)
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+- With the files open in Visual Studio Code, press the **Go Live** button at the bottom of the window to launch the files with [Live Server](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer).
```

Comment:
[ OPTIONAL CHANGES ] But I highly recommend you change them

I believe you used a previous project README here, but to start a react app, which runs on a server you go through these steps
- Clone the reop
- cd into the project directory
- npm install
- npm start (to start/run the project)

I kindly recommend you write those steps instead of the ones you have currently. 👍